rubygem-foreman_maintain/rubygem-foreman_maintain.spec

```diff
@@ -2,14 +2,14 @@
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name foreman_maintain
-%global confdir foreman_maintain
+%global package_name foreman-maintain
 
 %{!?_root_bindir:%global _root_bindir %{_bindir}}
 
 Summary: The Foreman/Satellite maintenance tool
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.0.3
-Release: 1%{?dist}
+Version: 0.0.6
+Release: 2%{?dist}
```

Comment:
While updating the Release to 1, bump the version number to 0.0.7 as well please, as I've just pushed the new version to rubygems: it includes the the full upgrade path work.